helpers/lib/common.js

```diff
@@ -10,21 +10,21 @@ function isValidURL(val) {
 }
 
 /*
- * Based on https://github.com/jonschlinkert/get-value/blob/3.0.1/index.js, but
```

Comment:
What was the intend to update this library? Looks like 2.0.6 is the current version in handlebars helpers.
Did you check if there are any breaking changes in behaviour?